static/src/javascripts/projects/common/modules/experiments/tests/contributions-epic.js

```diff
@@ -0,0 +1,88 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-epic.html',
```

Comment:
So for now, I do not believe that it is worth creating a multipurpose template as one of the tempaltes we would be combining is actually going to be retired very soon, possibly today or tomorrow (it's the giraffe-message.html). This is because the results show it isn't as effective as the other one. As we will be iterating on the contributions-epic.html template, what makes more sense is to make this more paramatised next time we iterate it.